server.py

```diff
@@ -5,13 +5,18 @@
 import logging
 import aiohttp
 import aiohttp.server
+import asyncio_redis
 import dorks_manager
 import session_manager
 import api
 import base_handler
 import logger
+import uvloop
 
+asyncio.set_event_loop_policy(uvloop.EventLoopPolicy())
 logger = logger.Logger.create_logger('tanner.log', 'tanner')
+redis_client = asyncio.get_event_loop().run_until_complete(
+    asyncio_redis.Pool.create(host='localhost', port=6379, poolsize=10))
```

Comment:
It was the most tricky question for me - how to organize new asyncio redis. Before I initialize redis for every emulator in init, but I can't use generators in __init__. This works, but maybe there is more elegant (and right) way?